### PR TITLE
feat: support pass custom backend options

### PR DESCRIPTION
**Pull request by percy507**
In consideration of the issue: [HTML5 Backend breaks all inputs text drag](https://github.com/react-dnd/react-dnd/issues/1552), I think we should support the user can custom the backend options.

### CODE REVIEW

**Comment by minop1205** [comment]
@percy507 Thank you! Please give me some time to check.

File: src/index.ts [hunk 1]
```diff
@@ -7,4 +7,4 @@ export * from "./Tree";
 export * from "./types";
 export * from "./utils";
 export { MultiBackend } from "dnd-multi-backend";
-export { HTML5toTouch } from "./HTML5toTouch";
+export { getBackendOptions } from "./HTML5toTouch";
```
Comment (minop1205) [inline src/index.ts]: export { getBackendOptions } from "./getBackendOptions";

File: src/HTML5toTouch.ts [hunk 2]
```diff
@@ -1,25 +1,31 @@
...
-    },
-    {
-      id: "touch",
-      backend: TouchBackend,
-      options: { enableMouseEvents: true },
-      preview: true,
-      transition: TouchTransition,
-    },
-  ],
+export const getBackendOptions = (
```
Comment (minop1205) [inline src/HTML5toTouch.ts]: Rename the file `getBackendOptions.ts`.

**Comment by minop1205** [comment]
Confirmed, I was able to work around the problem by setting the scope of react-dnd in rootElement.

I have one request, since `src/HTML5toTouch.ts` is now a function, could you please change the file name to `src/getBackendOptions.ts` accordingly?

I would like to export the following in `src/index.ts`.

```
export { getBackendOptions } from ". /getBackendOptions";
```

File: src/HTML5toTouch.ts (hunk 2 above)
Comment (percy507) [reply src/HTML5toTouch.ts]: Do you mind if I move the file into utils folder after renaming?

Comment (minop1205) [reply src/HTML5toTouch.ts]: Yes, it is! I think it's better that way!

File: src/index.ts (hunk 1 above)
Comment (minop1205) [reply src/index.ts]: If you are migrating to utils, this line may not be necessary.

**Comment by percy507** [comment]
OK. The job is done.

**Comment by minop1205** [comment]
Thank you for your response. Merged!

**Pull request merged**